server/views/pages/adjustments/hub.njk

```diff
@@ -95,6 +95,10 @@
             </ol>
           {% elif model.unusedDeductionMessage == 'NOMIS_ADJUSTMENT' %}
             Existing deductions have been added on NOMIS. This means unused deductions cannot be automatically calculated. To add any unused remand, go to the sentence adjustments screen in NOMIS.
+          {% elif model.unusedDeductionMessage == 'CRDS_CALL_FAILED' %}
+            Some of the details recorded cannot be used for a sentence calculation. This means unused deductions cannot be automatically calculated by this service. You can <a href="#">edit or delete the unused deductions here.</a>
```

Comment:
You have two choices for the # link. You can either continue on this branch and implement the next page. Or you can feature toggle this functionality. You can't really leave a # link like this on a live system.